modules/release-notes/pages/index.adoc

```diff
@@ -169,6 +169,24 @@ See xref:3.10@tigergraph-server:user-access:rbac-row-policy/row-policy-privilege
 
 == Fixed issues
 
+=== Fixed and Improved in 3.10.4
+
+==== Functionality
+
+* Fixed an issue where the `/codecheck` endpoint returned an empty response for queries with syntax or semantic errors, which previously allowed incorrect queries to be installed (GLE-10432).
+* Fixed a backward compatibility issue that caused system unavailability or schema check failures after upgrading when the keyword `function` was used in vertex attributes (GLE-9840).
+* Fixed an *Out-of-Memory (OOM)* issue that occurred during file loading operations (TP-8151).
+* Fixed an issue where vertex attributes loaded with invalid values when a loading job inserted vertices using reducer operators such as `and()`, `ignore_if_exists()`, `max()`, or `min()` (CORE-4866).
+
+==== Improvements
+
+* Upgraded Nginx to version 1.26 to enhance compliance and provide an updated underlying service component (TP-8129).
```

Comment:
In general, you don't need to explain why we upgrade components. Only if there was a specific new feature (in Nginx) we used or a specific bug (in Nginx) that was fixed.

Otherwise, just say "Upgraded Nginx to version 1.26."